regression_filter.rb

```diff
@@ -0,0 +1,18 @@
+require 'rspec'
+require 'watir'
+require 'helper'
+
+describe 'Regression Test Cases: ' do
+  let(:method) { @main.methods }
+  let(:regression_method) { @main.regression_methods }
+
+  filter_categories_data = {
```

Comment:
filter_categories_data could be expanded for : Technology,  Jewelry and Collectibles as well in the same way.